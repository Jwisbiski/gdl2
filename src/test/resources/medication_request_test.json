{
  "resourceType": "MedicationRequest",
  "status": "draft",
  "intent": "{$gt0000}",
  "medicationCodeableConcept": {
    "coding": [
      {
        "system": "{$gt0001}",
        "code": "{$gt0002}",
        "display": "{$gt0003}"
      }
    ]
  },
  "dosageInstruction": [
    {
      "sequence": "{$gt0004}",
      "text": "{$gt0005}{$gt0006} once daily",
      "timing": {
        "repeat": {
          "frequency": "{$gt0007}",
          "period": "{$gt0008}",
          "periodUnit": "{$gt0009}"
        }
      },
      "doseQuantity": {
        "value": "{$gt0005}",
        "unit": "{$gt0006}",
        "system": "{$gt0010}",
        "code": "{$gt0006}"
      }
    }
  ]
}