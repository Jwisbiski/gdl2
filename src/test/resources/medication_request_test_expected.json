{
  "resourceType": "MedicationRequest",
  "status": "draft",
  "intent": "plan",
  "medicationCodeableConcept": {
    "coding": [
      {
        "system": "http://www.whocc.no/atc",
        "code": "C10AA05",
        "display": "atorvastatin"
      }
    ]
  },
  "dosageInstruction": [
    {
      "sequence": 1,
      "text": "7.0mg once daily",
      "timing": {
        "repeat": {
          "frequency": 1,
          "period": 1,
          "periodUnit": "d"
        }
      },
      "doseQuantity": {
        "value": 7,
        "unit": "mg",
        "system": "http://unitsofmeasure.org",
        "code": "mg"
      }
    }
  ]
}